.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6-node
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "requirements.txt" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies
```

Comment:
This is neat. I figured the keys were just unique strings and not tiered values.

Reply:
Yeah that's just from Circle's boilerplate. sounded nice to me!